atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -111,6 +111,7 @@ const CreateCrashRecord = ({ client }) => {
               sus_serious_injry_cnt: $sus_serious_injry_cnt
               temp_record: true
               crash_date: $crash_date
+              city_id: 22  
```

Comment:
Going to make sure this is an editable field with a variable that gets passed into the query when I make updates to the form.